soccer/gameplay/standard_play.py

```diff
@@ -34,9 +38,24 @@ def use_standard_defense(self):
     #method in additionto having an "execute_running" method of its own must call
     #it via super
     def execute_running(self):
+        main.situationAnalysis.currentPreempt
+        if (main.situationAnalysis.currentPreempt):
+            self.terminate()
+
         self.use_standard_defense()
 
     #Since the standard_play handles defense, it will always handle the goalie
     @classmethod
     def handles_goalie(cls):
         return True
+
+    @classmethod
+    def score(cls):
```

Comment:
I really like this.

Reply:
What about it, exactly?

Reply:
Also, we need to have a lot more documentation about what the overridden class method needs to do.